docs/encyclopedia/temporal-nexus-endpoints.mdx

```diff
@@ -0,0 +1,27 @@
+---
+id: nexus-endpoints
+title: Nexus Endpoints
```

Comment:
There's a lot of content for the Nexus Operation that we should get in esp. for the Operation Lifecycle, Automatic Retries, Execution Semantics, and Versioning.